group_vars/arbeitsplatz.yml

```diff
@@ -1,8 +1,8 @@
+postfix_host_email: gst-clients@hamburg.adfc.de
```

Comment:
Sollten wir diese Adresse ebenfalls aus github raushalten, und in eine .sec Datei verschieben?